health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FormulaConfigWrapper.js

```
@@ -1,42 +1,441 @@
-    const { state, dispatch } = useMyContext();
+   const { mutate: updateResources, ...rest } = Digit.Hooks.microplanv1.useCreateUpdatePlanProject();
+   const { t } = useTranslation();
+   const { state} = useMyContext();
+   const [formulaConfigValues, setFormulaConfigValues] = useState(customProps?.sessionData?.FORMULA_CONFIGURATION?.formulaConfiguration?.formulaConfigValues || [])
+   const assumptionsFormValues = customProps?.sessionData?.ASSUMPTIONS_FORM?.assumptionsForm //array with key value pair
+   const campaignType =  customProps?.sessionData?.CAMPAIGN_DETAILS?.campaignDetails?.campaignType?.code
+   const resourceDistributionStrategyCode= customProps?.sessionData?.CAMPAIGN_DETAILS?.campaignDetails?.distributionStrat?.resourceDistributionStrategyCode
+   const searchParams = new URLSearchParams(window.location.search);
+   const [formulaInternalKey, setFormulaInternalKey] = useState(() => {
+      const keyParam = searchParams.get("formulaInternalKey");
+      return keyParam ? parseInt(keyParam) : 1;
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)